IbisHardwareIGSIO/ibishardwareIGSIO.cpp

```diff
@@ -386,16 +386,42 @@ void IbisHardwareIGSIO::InitPlugin()
 {
```

Comment:
If you do it like this in the InitPlugin() function, changes to the path in the preference dialog will not be taken into account until you restart Ibis.